### ISSUE TITLE
Managed automations

### ISSUE THREAD

**Issue by jmduke** [buttondown/roadmap#2966]
## TL;DR

Rather than have automations just be a blank canvas, we should also offer _managed_ automations, which are pre-canned automations that we've built. Managed automations can be used regardless of your billing plan, but cannot be customized.

- [ ] Expose `Automation.source` in the API
- [ ] Tweak billing logic on the frontend and backend to respect `Automation.source`
- [ ] Move the automations created by `Autonomy` to the backend, and have them created as 'managed'
- [ ] Create a managed automation for `email.sent` to `LinkedIn`
- [ ] Create some sort of catalog/browsing interface for all managed automations
- [ ] Surface managed automations in the marketing site
- [ ] Unship `specific_email_to_send_to_new_subscribers_id`

## Prior art

https://ifttt.com/applets/DfUKrQkt-automatically-save-in-pocket-the-first-link-in-a-tweet-you-like
